pkg/apis/serving/v1alpha1/route_types.go

```diff
@@ -88,6 +88,17 @@ type TrafficTarget struct {
 	Percent int `json:"percent"`
 }
 
+const (
```

Comment:
I would prefer these go in service_types.go as they are set using when using release mode in Service. The names are really a property of the mode and not a property of the route. 

Otherwise everything else looks good.

Reply:
Works for me.
Done.